packages/Discretization/test/PointCloudProblemGenerator/create_ascii_data_from_exodus.py

```diff
@@ -0,0 +1,71 @@
+# This script reads and exodus file and dumps its node coordinates and
```

Comment:
Missing copyright blurb + typo and -> an

Reply:
good catch